source/swap/contracts/Swap.sol

```diff
@@ -335,10 +337,12 @@ contract Swap is ISwap {
   ) internal {
     if (kind == ERC721_INTERFACE_ID) {
       // Attempt to transfer an ERC-721 token.
-      IERC721(token).safeTransferFrom(from, to, param);
+      IERC721(token).transferFrom(from, to, param);
```

Comment:
Do we need to document this somewhere so that the frontend check gets picked up by the frontend devs?

Reply:
yes, jotting these down in the issue + notion doc and will share